docs/faq/security.mdx

```diff
@@ -0,0 +1,20 @@
+---
+id: security 
+hide_title: true
+title: Security FAQ
+sidebar_label: Security
+slug: /faq/security
+---
+
+# FAQ - Security
+
+The security of the network and Hotspot owners is of the utmost importance to us.  While the internet connection is necessary, it is only used to provide backhaul to the Hotspot for blockchain peer-to-peer and LongFi networking. As security concerns are ever present, here is what we have done, and are doing, to combat potential risks:
+
+- The most common attack vector to get into a device like the Hotspot is inbound ports.  The Helium Hotspot only requires one port open in both directions (TCP Port 44158), so all other inbound ports can be secured behind a firewall per your personal security needs.  
+- Helium devices are hardware secured to protect the traffic from the utilized spectrum.  This means the security is built-in since devices using the network have AES private key encryption at the chip level.  
```

Comment:
```suggestion
- Helium-enabled LoRaWAN devices are hardware-secured to protect the traffic from the utilized spectrum.  This means the security is built-in since devices using the network have AES private key encryption at the chip level.  
```